README.md

```diff
@@ -379,6 +382,11 @@ however disabled by default, and the generated errors are correctly handled by P
 ### 0.18
 
 * First public release
+</details>
+
+## Issues
+
+Issues management is described [here](ISSUES.md).
```

Comment:
Only one click away from [Don't use "click here" as link text](https://www.w3.org/QA/Tips/noClickHere).

Reply:
:sweat_smile: I rephrased it : what do you think about the new version ?